packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -139,9 +139,16 @@ class UserMessages {
 
   // Messages used in XcodeValidator
   String xcodeLocation(String location) => 'Xcode at $location';
-  String xcodeOutdated(int versionMajor, int versionMinor, int versionPatch) =>
-      'Flutter requires a minimum Xcode version of $versionMajor.$versionMinor.$versionPatch.\n'
```

Comment:
I figured there's no point mentioning the minimum required if they are out of date: just download the recommended one.